### PR TITLE
Organize the content pages server to be served a webapp, and share styles and assets with the browser for consistency

### PR DESCRIPTION
**Pull request by victorporof**
Preliminary refactoring before actually working on changing the content pages to use react. Depends on https://github.com/mozilla/tofino/pull/573

Relevant changes start with commit c858926 and down.

### CODE REVIEW

File: app/services/content-service/server.js [hunk 1]
```diff
@@ -10,14 +10,25 @@
  specific language governing permissions and limitations under the License.
  */
 
-import express from 'express';
 import path from 'path';
-import { makeServer } from '../common';
+import express from 'express';
```
Comment (Mossop) [inline app/services/content-service/server.js]: Avoiding reordering is good as it breaks blame

File: app/services/content-service/server.js [hunk 2]
```diff
@@ -10,14 +10,25 @@
...
+import { makeServer, autoCaughtRouteError } from '../common';
 import * as endpoints from '../../shared/constants/endpoints';
 import { BUILT_UI_DIR } from '../../shared/paths-util';
 
+const VALID_PAGES = /(history|stars)/;
```
Comment (Mossop) [inline app/services/content-service/server.js]: You need to add credits to this.

File: app/shared/constants/content-pages-locations.js [hunk 3]
```diff
@@ -12,6 +12,6 @@ specific language governing permissions and limitations under the License.
 
 import * as endpoints from './endpoints';
 
-export const HISTORY_PAGE = `${endpoints.CONTENT_SERVER_HTTP}/history.html`;
-export const STARS_PAGE = `${endpoints.CONTENT_SERVER_HTTP}/stars.html`;
+export const HISTORY_PAGE = `${endpoints.CONTENT_SERVER_HTTP}/history`;
+export const STARS_PAGE = `${endpoints.CONTENT_SERVER_HTTP}/stars`;
 export const CREDITS_PAGE = `${endpoints.CONTENT_SERVER_HTTP}/credits.html`;
```
Comment (Mossop) [inline app/shared/constants/content-pages-locations.js]: And drop .html from this

**Comment by Mossop** [comment]
LGTM

File: app/services/content-service/server.js (hunk 1 above)
Comment (victorporof) [inline app/services/content-service/server.js]: Accidental change, will revert.

**Pull request merged**